cmd/xl-storage-disk-id-check.go

```diff
@@ -426,3 +574,12 @@ func (p *xlStorageDiskIDCheck) ReadAll(ctx context.Context, volume string, path
 
 	return p.storage.ReadAll(ctx, volume, path)
 }
+
+// Update storage metrics
+func (p *xlStorageDiskIDCheck) updateStorageMetrics(s storageMetric) func() {
+	startTime := time.Now()
+	return func() {
+		atomic.AddUint64(&p.apiCalls[s], 1)
```

Comment:
Should the read be protected via `atomic`?

Reply:
read phase uses atomic.LoadUint64() @kerneltime